docs/format.md

```diff
@@ -527,3 +528,22 @@ recommendations when writing paragraph text and incorporating links.
 
 Model summaries, descriptions, and vocabulary entry descriptions can be
 translated. Please see [translation.md](./translation.md) for details.
+
+## Checking if everything is ok
+
+Use [spec-parser][] to check if your additions or edits are correctly
+formatted.
+
+Install:
+
+```shell
+git clone https://github.com/spdx/spec-parser.git
```

Comment:
If the intent is to only run it, cloning the main branch and without any history would be faster.

Reply:
Thank you. Added the branch and depth options.